pkg/manager/integration/manager_suite_test.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestSource(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Manager Integration Suite")
+}
```

Comment:
Seems the tests aren't running yet:

```
Running Suite: Manager Integration Suite - /home/prow/go/src/sigs.k8s.io/controller-runtime/pkg/manager/integration
===================================================================================================================
Random Seed: 1690107923
Will run 0 of 0 specs
```

Reply:
Now it works. Had to change one Describe to It

```
Running Suite: Manager Integration Suite - /home/prow/go/src/sigs.k8s.io/controller-runtime/pkg/manager/integration
===================================================================================================================
Random Seed: [1m1690219195[0m

Will run [1m1[0m of [1m1[0m specs
{"level":"info","ts":"2023-07-24T17:20:05Z","logger":"controller-runtime.webhook","msg":"Registering webhook","path":"/convert"}
{"level":"info","ts":"2023-07-24T17:20:05Z","msg":"Starting server","kind":"health probe","addr":"[::]:40593"}
{"level":"info","ts":"2023-07-24T17:20:05Z","logger":"controller-runtime.webhook.webhooks","msg":"Starting webhook server"}
{"level":"info","ts":"2023-07-24T17:20:05Z","logger":"controller-runtime.certwatcher","msg":"Updated current TLS certificate"}
{"level":"info","ts":"2023-07-24T17:20:05Z","logger":"controller-runtime.certwatcher","msg":"Starting certificate watcher"}
{"level":"info","ts":"2023-07-24T17:20:05Z","logger":"controller-runtime.webhook","msg":"Serving webhook server","host":"127.0.0.1","port":43437}
W0724 17:20:05.282787   27419 reflector.go:535] pkg/mod/k8s.io/client-go@v0.28.0-alpha.3/tools/cache/reflector.go:229: failed to list *v2.Driver: conversion webhook for crew.example.com/v1, Kind=Driver failed: Post "https://127.0.0.1:43437/convert?timeout=30s": dial tcp 127.0.0.1:43437: connect: connection refused
E0724 17:20:05.282918   27419 reflector.go:147] pkg/mod/k8s.io/client-go@v0.28.0-alpha.3/tools/cache/reflector.go:229: Failed to watch *v2.Driver: failed to list *v2.Driver: conversion webhook for crew.example.com/v1, Kind=Driver failed: Post "https://127.0.0.1:43437/convert?timeout=30s": dial tcp 127.0.0.1:43437: connect: connection refused
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Starting EventSource","controller":"driver","controllerGroup":"crew.example.com","controllerKind":"Driver","source":"kind source: *v2.Driver"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Starting Controller","controller":"driver","controllerGroup":"crew.example.com","controllerKind":"Driver"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Starting workers","controller":"driver","controllerGroup":"crew.example.com","controllerKind":"Driver","worker count":1}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Reconciling","controller":"driver","controllerGroup":"crew.example.com","controllerKind":"Driver","Driver":{"name":"driver1","namespace":"default"},"namespace":"default","name":"driver1","reconcileID":"39f239d4-e743-49b0-9485-c18de1bcbac1"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Stopping and waiting for non leader election runnables"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Stopping and waiting for leader election runnables"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Shutdown signal received, waiting for all workers to finish","controller":"driver","controllerGroup":"crew.example.com","controllerKind":"Driver"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"All workers finished","controller":"driver","controllerGroup":"crew.example.com","controllerKind":"Driver"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Stopping and waiting for caches"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Stopping and waiting for webhooks"}
{"level":"info","ts":"2023-07-24T17:20:06Z","logger":"controller-runtime.webhook","msg":"Shutting down webhook server with timeout of 1 minute"}
{"level":"info","ts":"2023-07-24T17:20:06Z","msg":"Wait completed, proceeding to shutdown the manager"}
[38;5;10m•[0m

[38;5;10m[1mRan 1 of 1 Specs in 12.438 seconds[0m
[38;5;10m[1mSUCCESS![0m -- [38;5;10m[1m1 Passed[0m | [38;5;9m[1m0 Failed[0m | [38;5;11m[1m0 Pending[0m | [38;5;14m[1m0 Skipped[0m
--- PASS: TestSource (12.44s)
PASS
ok  	sigs.k8s.io/controller-runtime/pkg/manager/integration	12.600s
```